libs/services/designer-client-services/src/lib/search.ts

```diff
@@ -2,12 +2,14 @@ import type { Connector, DiscoveryOperation, DiscoveryResultTypes } from '@micro
 import { AssertionErrorCode, AssertionException } from '@microsoft/utils-logic-apps';
 
 export interface ISearchService {
-  search(term: string): Promise<SearchResult>;
   getAllConnectors(): Promise<Connector[]>;
   getAzureConnectorsByPage(page: number): Promise<Connector[]>;
   getCustomConnectorsByNextlink(nextlink?: string): Promise<{ nextlink?: string; value: Connector[] }>;
   getBuiltInConnectors(): Promise<Connector[]>;
   getAllOperations(): Promise<DiscoveryOperation<DiscoveryResultTypes>[]>;
+  getRequestWorkflows(): Promise<any>;
+  getBatchWorkflows(): Promise<any>;
+  getWorkflowTriggers(workflowId: string): Promise<any[]>;
```

Comment:
Is it possible to strongly type these? (We are trying to avoid use of `any` wherever possible, even in interface implementations.)